app/src/main/java/com/kickstarter/ui/viewholders/discoverydrawer/LoggedInViewHolder.kt

```diff
@@ -0,0 +1,77 @@
+package com.kickstarter.ui.viewholders.discoverydrawer
+
+import android.view.View
+import androidx.annotation.NonNull
+import com.kickstarter.libs.rx.transformers.Transformers.observeForUI
+import com.kickstarter.libs.transformations.CircleTransformation
+import com.kickstarter.libs.utils.IntegerUtils
+import com.kickstarter.libs.utils.NumberUtils
+import com.kickstarter.libs.utils.ViewUtils
+import com.kickstarter.models.User
+import com.kickstarter.ui.viewholders.KSViewHolder
+import com.kickstarter.viewmodels.LoggedInViewHolderViewModel
+import com.squareup.picasso.Picasso
+import kotlinx.android.synthetic.main.discovery_drawer_logged_in_view.view.*
+
+class LoggedInViewHolder(@NonNull view: View, @NonNull private val delegate: Delegate) : KSViewHolder(view) {
```

Comment:
Shouldn't this class be `final`?

Reply:
Nope!
![Screen Shot 2019-05-28 at 10 17 31 AM](https://user-images.githubusercontent.com/1289295/58485353-df39d280-8131-11e9-83d1-02d42f307fdb.png)